tests/Application/composer.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "sylius/plugin-skeleton-test-application",
```

Comment:
Not sure we need this file here

Reply:
Yes, it is used to deduce the project dir as written in the description of the same file: `Sylius application for plugin testing purposes (composer.json needed for project dir resolving)`